client/landing/stepper/declarative-flow/plugin-bundle-flow.ts

```diff
@@ -129,11 +130,16 @@ const pluginBundleFlow: Flow = {
 		function submit( providedDependencies: ProvidedDependencies = {}, ...params: string[] ) {
 			recordSubmitStep( providedDependencies, intent, flowName, currentStep );
 
+			const defaultExitDest = siteDetails?.options
+				? `/theme/${ siteDetails?.options.theme_slug }/${ siteSlug }`
```

Comment:
Is `siteDetails?.options.theme_slug` guaranteed to be defined? The `?.` doesn't inspire confidence!